tailwind.config.js

```diff
@@ -1,10 +1,44 @@
+const { colors } = require('tailwindcss/defaultTheme');
+
 module.exports = {
-  prefix: "tw-",
+  prefix: 'tw-',
   theme: {
     extend: {
       colors: {
-        "dbt-orange": "#ff694b",
+        ...colors,
+        transparent: {
+          default: 'var(--color-transparent)',
+        },
+        white: {
+          ...colors.white,
+          default: 'var(--color-white)',
```

Comment:
Just out of curiosity – is there a benefit to referencing these like this instead of just putting the values in this file?

Reply:
This gives us a few benefits -
- We can use the css variables in JS if we need to something dynamic/on the fly and still use the theme colors
- We can use the css variables in places where there isn't tailwind support/we haven't added tailwind support
  - (ex. fill colors on svgs)
- This makes theming as easy as redefining the css variables, if we want to change themes on the fly later